Purchases/SwiftSources/LocalReceiptParsing/BasicTypes/ASN1Container.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Created by Andrés Boedo on 7/28/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum ASN1Class: UInt8 {
+    case universal, application, contextSpecific, `private`
+}
+
+enum ASN1Identifier: UInt8, CaseIterable {
+    case endOfContent = 0x00
```

Comment:
Does this have to be `0x00` or can it be just `0`

Reply:
good catch